pkgs/applications/window-managers/notion/default.nix

```diff
@@ -1,39 +1,32 @@
 {
-  enableXft ? true, libXft ? null, patches ? [], stdenv, lua, gettext, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
-  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which
+  enableXft ? true, libXft ? null, stdenv, lua, gettext, readline, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
+  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which, groff
 }:
 
 assert enableXft -> libXft != null;
 
 let
   pname = "notion";
-  version = "3-2017050501";
-  inherit patches;
+  version = "4.0.git";
```

Comment:
We're not Arch, this derivation doesn't build from the latest git revision :laughing:
Please add the date of the commit, like described in https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
Yeah, I'll update this once we actually release 4.0.0 and I un-draft this PR ;)

But thanks for the reference, good to have even in-progress branches follow all the right conventions :)